### PR TITLE
CCTRI-1650 Changed delimiter from dot to colon

### PR DESCRIPTION
**Pull request by mmysk**

### CODE REVIEW

File: api/client.py [hunk 1]
```diff
@@ -43,7 +43,9 @@ def query(self, endpoint, headers=None, params=None):
             return None
 
         if response.status_code == HTTPStatus.FORBIDDEN:
-            raise AuthenticationRequiredError
+            raise AuthenticationRequiredError(
+                reason="Authorization failed on <3rd party name> side"
+            )
```
Comment (GeorgiyII) [inline api/client.py:48]: Is there any error text in AlienVault response?
If not, I think will be better: Authorization failed on AlienVault side

File: api/errors.py [hunk 2]
```diff
@@ -29,7 +29,18 @@ class InvalidPayloadReceivedError(RelayError):
...
+    def __init__(self, reason=None):
+        message = self.MESSAGE
+        if reason:
+            message += f': {reason}'
+        else:
+            message += "."
+
+        super().__init__(message)
+
+        self.message = message
```
Comment (GeorgiyII) [inline api/errors.py:43]: why you repeated the code, just fix RelayError i think

File: api/client.py (hunk 1 above)
Comment (mmysk) [reply api/client.py]: Done

File: api/errors.py (hunk 2 above)
Comment (mmysk) [reply api/errors.py:43]: Because of different error message style

**Review by GeorgiyII — APPROVED** [review]

**Review by nonspher — APPROVED** [review]

**Pull request merged**